content/tutorial/12-errors.md

```diff
@@ -4,17 +4,17 @@ date: 2021-02-17T12:49:32Z
 description: "Okay, we've made it to part 12! Be forewarned, this is going a long one. This part is going to be about how you can make error handlers for your commands, like you saw briefly in the previous part about cooldowns."
 ---
 
-There are several of ways you can go about error handling with discord.py:
+There are several ways you can go about error handling with discord.py:
 
 - Per-command error handlers
 - Per-cog error handlers
 - Global error handlers
 
-In this part I will only be covering per-command and global error handlers, but by all means feel free to go down the per-cog route by reading [the docs](https://discordpy.readthedocs.io/) and experimenting on your own. Again, as a general reminder I advise you don't just copy the code here, rather try to understand it yourself and make your own code. You don't learn by copying, you learn by doing.
+In this part, I will only be covering per-command and global error handlers, but by all means, feel free to go down the per-cog route by reading [the docs](https://discordpy.readthedocs.io/) and experimenting on your own. Again, as a general reminder, I advise you don't just copy the code here, rather try to understand it yourself and make your code. You don't learn by copying, you learn by doing.
```

Comment:
I'd rather keep the `own` in here - it may not be quite correct, but it doesn't fit my writing style or sound right to me if it's removed, this applies to other cases where it's removed which I'll put a small note on too